streams/next.json

```diff
@@ -1,92 +1,92 @@
 {
     "stream": "next",
     "metadata": {
-        "last-modified": "2022-04-12T18:02:21Z",
-        "generator": "fedora-coreos-stream-generator v0.2.5"
+        "last-modified": "2022-04-19T02:09:15Z",
+        "generator": "fedora-coreos-stream-generator v0.2.6"
```

Comment:
This is flagged by CI as an unexpected change, but I think it's normal. Ignore?

Reply:
https://github.com/coreos/fedora-coreos-streams/pull/491 changes the stream diff workflow to avoid warning on these changes if they're expected.